apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -13,21 +12,17 @@ interface CreateProjectCommand {
 export async function createProject({ resource, store }: CreateProjectCommand): Promise<GraphPointer> {
   const label = resource.out(rdfs.label).term!
 
-  const project = clownface({ dataset: $rdf.dataset() })
-    .namedNode(id.cubeProject(label.value))
+  const project = store.create(cubeProject(label.value))
     .addOut(rdfs.label, label)
     .addOut(rdf.type, [cc.CubeProject, hydra.Resource])
 
   if (resource.out(cc.projectSourceKind).value === 'CSV') {
-    const csvMapping = clownface({ dataset: $rdf.dataset() })
-      .namedNode(id.csvMapping(project))
+    const mapping = store.create(csvMapping(project))
       .addOut(rdf.type, [cc.CsvMapping, hydra.Resource])
 
-    project.addOut(cc.csvMapping, csvMapping)
-
-    await store.put(csvMapping)
+    project.addOut(cc.csvMapping, mapping)
   }
 
-  await store.put(project)
+  await store.save()
```

Comment:
And everything will be saved in one go, without doing multiple roundtrips to the SPARQL graph endpoint